src/puptoo/utils/config.py

```diff
@@ -51,6 +51,8 @@ def get_namespace():
     AWS_ACCESS_KEY = os.getenv("STORAGE_ACCESS_KEY", LoadedConfig.objectStore.buckets[0].accessKey)
     AWS_SECRET_KEY = os.getenv("STORAGE_SECRET_KEY", LoadedConfig.objectStore.buckets[0].secretKey)
     USE_SSL = os.getenv("USE_SSL", True)
+    REDIS_HOST = LoadedConfig.inMemoryDb.hostname
+    REDIS_PORT = LoadedConfig.inMemoryDb.port
```

Comment:
@r14chandra I think we also need to get `username` and `password` from clowder config like this - https://github.com/RedHatInsights/ros-backend/blob/main/ros/lib/config.py#L50-L51

Reply:
@r14chandra had a chance to look back into this?

Reply:
@csams It would work anyway, right? As I don't see insights-engine has added this config.

Reply:
I believe it should work without this - an easy way to test would be to run it in the ephemeral environments with a test in the pr_check and see what happens